qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -472,10 +521,17 @@ bool HOCRPdfExporter::run(QString& filebasename) {
 				bool success = false;
 				QMetaObject::invokeMethod(this, "setSource", Qt::BlockingQueuedConnection, Q_RETURN_ARG(bool, success), Q_ARG(QString, page->sourceFile()), Q_ARG(int, page->pageNr()), Q_ARG(int, outputDpi), Q_ARG(double, page->angle()));
 				if(success) {
-					double docScale = (72. / sourceDpi);
+					double docScale = (72.0 / sourceDpi);
 					double imgScale = double(outputDpi) / sourceDpi;
-					PoDoFo::PdfPage* pdfpage = document->CreatePage(PoDoFo::PdfRect(0, 0, bbox.width() * docScale, bbox.height() * docScale));
-					pdfprinter.setPage(pdfpage, docScale);
+					PoDoFo::PdfPage* pdfpage;
+					if(paperSize == "source") {
+                        width = bbox.width() * docScale;
+                        height = bbox.height() * docScale;
+					}
+					pdfpage = document->CreatePage(PoDoFo::PdfRect(0, 0, width, height));
+                    double offsetX = 0.5 * (width - bbox.width() * docScale);
+                    double offsetY = 0.5 * (height - bbox.height() * docScale);
```

Comment:
wrong indentation (fix your editor ;))

Reply:
Just launched astyle. Fixed (sorry for this).